generic/tests/ping.py

```diff
@@ -9,8 +10,12 @@
 def _ping_with_params(test, params, dest, interface=None,
                       packet_size=None, interval=None,
                       count=0, session=None, flood=False):
+
+    if os.getuid() != 0 and not interval:
```

Comment:
If the argument `session` is given, the following ping command would be sent to the VM. In this case it should get the `uid` from the VM.